report/verification/mes/tmap_1c.md

```diff
@@ -0,0 +1,128 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.2
+kernelspec:
+  display_name: vv-festim-report-env
+  language: python
+  name: python3
+---
+
+# TMAP7 V&V Val-1c
+
+```{tags} 1D, MES, transient
+```
+
+This verification case {cite}`ambrosek_verification_2008` from TMAP7's V&V document consists of a semi-infinite slab with no traps under a constant concentration $C_0$ on the first $10 m$ of the slab.
+
++++
+
+## FESTIM Code
+
+```{code-cell} ipython3
+:tags: [hide-cell]
+
+import festim as F
+import numpy as np
+import sympy as sp
+from matplotlib import pyplot as plt
+
+preloaded_length = 10 # m
+C_0 = 1 # atom m^-3
+D = 1 # 1 m^2 s^-1
+exact_solution = C_0/2 * (
+    2*sp.erf(F.x / sp.sqrt(4*D*F.t)) 
+    - sp.erf((F.x - preloaded_length) / sp.sqrt(4*D*F.t))
+    - sp.erf((F.x + preloaded_length) / sp.sqrt(4*D*F.t))
+)
```

Comment:
Instead of using a sympy expression I would do the same refactoring I did in #53